<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="1"
            failures="0"
            tests="1">

  <testsuite name="localhosttest"
    errors="1" failures="0" hostname="" id="" package="localhosttest" skipped="0"
    tests="1" time="0.03300" timestamp="Thu, 04 Aug 2016 18:15:51 GMT">
  
    <testcase name="step one" time="0.03300" assertions="0">
    </testcase>
      <system-err>
          Error while running [Localhosttest / step one]:

  TypeError: undefined is not a function
    at Object.module.exports.step one (/Users/kelli/Sites/nightwatch/test/demotests/localhosttest.js:7:8)
    at Module.call (/Users/kelli/.nvm/versions/node/v0.12.7/lib/node_modules/nightwatch/lib/runner/module.js:62:34)
    at /Users/kelli/.nvm/versions/node/v0.12.7/lib/node_modules/nightwatch/lib/runner/testcase.js:70:29
    at _fulfilled (/Users/kelli/.nvm/versions/node/v0.12.7/lib/node_modules/nightwatch/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/Users/kelli/.nvm/versions/node/v0.12.7/lib/node_modules/nightwatch/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/Users/kelli/.nvm/versions/node/v0.12.7/lib/node_modules/nightwatch/node_modules/q/q.js:796:13)
    at /Users/kelli/.nvm/versions/node/v0.12.7/lib/node_modules/nightwatch/node_modules/q/q.js:556:49
    at runSingle (/Users/kelli/.nvm/versions/node/v0.12.7/lib/node_modules/nightwatch/node_modules/q/q.js:137:13)
    at flush (/Users/kelli/.nvm/versions/node/v0.12.7/lib/node_modules/nightwatch/node_modules/q/q.js:125:13)
    at process._tickCallback (node.js:355:11)
      </system-err>
  
  
  </testsuite>
</testsuites>
